src/basic/random-util.c

```diff
@@ -42,14 +42,14 @@ static void fallback_random_bytes(void *p, size_t n) {
                 uint8_t auxval[16];
         } state = {
                 /* Arbitrary domain separation to prevent other usage of AT_RANDOM from clashing. */
```

Comment:
This comment should probably be moved along?

Reply:
To the whole struct. I think it's fine as is.